test/functional/ex_cmds/print_commands_spec.lua

```diff
@@ -0,0 +1,11 @@
+local helpers = require('test.functional.helpers')(after_each)
+local eq, command, funcs = helpers.eq, helpers.command, helpers.funcs
+
+describe(':z^', function()
+  before_each(clear)
+
+  it('correctly sets the cursor after :z^', function()
+    command('silent! z^')
```

Comment:
`z^` should not error out or print anything  under current circumstances if I am not mistaking. So it needs not be silenced.